rdflib/graph.py

```diff
@@ -1327,6 +1327,8 @@ def parse(
         parser = plugin.get(format, Parser)()
         try:
             # TODO FIXME: Parser.parse should have **kwargs argument.
+            if format not in ["application/ld+json", "json-ld"] and "resolver" in args:
+                del args["resolver"]
```

Comment:
I don't like this, as the kwargs passed to parse should not be given any special meaning by RDFLib, they are entirely for the plugins to use. While this may have some design flaws it is the way it works now and this breaks that.

Reply:
I'm going to remove this.